### ISSUE TITLE
Last Fable.React upgrade breaks my code

### ISSUE THREAD

**Issue by OnurGumus** [fable-compiler/fable-react#232]
<img width="919" alt="image" src="https://user-images.githubusercontent.com/4114592/189547158-6445a331-1ab7-4e49-8b0a-a3dace5dba8e.png">

I guess screenshot says all. Any ideas?

**Comment by OnurGumus** [comment]
I was using Lit sample by the way

**Comment by alfonsogarciacaro** [comment]
We're trying to split React types and helpers in two different libraries (see [discussion](https://github.com/Zaid-Ajaj/Feliz/issues/508)). The idea is we can have different React APIs (original React, Feliz, incoming JSX, etc..) to be compatible with each other without having to depend on the full Fable.React. But unfortunately this breaks code if you're depending on a library that hasn't been updated, because F# doesn't support type forwarding.

What are the other React dependencies in your project? In any case you probably don't need to update right now.

**Comment by OnurGumus** [comment]
I understand. In that case I will stick to older version. Feel free to close the issue if you will.

**Comment by kerams** [comment]
Does this mean we have to pin `~> 8` until every single library we're referencing has switched?

**Comment by kerams** [comment]
And conversely, if I create a new bindings package and use `Fable.React.Types`, I can't use it if I also reference packages that haven't switched?

**Comment by kerams** [comment]
Would it be possible to release a new beta of Elmish.React which uses Fable.React.Types, so that Fulma and its extensions could in turn upgrade to it as well? That would unblock the upgrade path to Fable.React 9 (and Fable 4) in my project at least.

cc @MangelMaxime

**Comment by alfonsogarciacaro** [comment]
> And conversely, if I create a new bindings package and use Fable.React.Types, I can't use it if I also reference packages that haven't switched?

Yes, this is very unfortunate because F# doesn't support type forwarding. The funny thing is it does work in Fable because we always compile the sources, but having errors in the IDE is not nice. Let's try to update all packages quickly so we can make the pain short-lived :)

> Would it be possible to release a new beta of Elmish.React which uses Fable.React.Types, so that Fulma and its extensions could in turn upgrade to it as well? That would unblock the upgrade path to Fable.React 9 (and Fable 4) in my project at least.

[Elmish.React has just been released](https://github.com/elmish/react/pull/68#issuecomment-1250965597) :tada:
